lib/facebook/messenger/subscriptions.rb

```diff
@@ -9,7 +9,7 @@ module Messenger
     module Subscriptions
       include HTTParty
 
-      base_uri 'https://graph.facebook.com/v3.2'
+      base_uri 'https://graph.facebook.com/v4.0'
```

Comment:
Seems this is the version, we are enforcing on the dev production app so I matched it here too.

Reply:
@anam-mansoor7

I know we have some vestiges of v8.0 but the developer app states v4.0: 
<img width="1605" alt="Screen Shot 2021-08-05 at 11 28 13" src="https://user-images.githubusercontent.com/15090945/128335457-62d73310-53cc-4a29-a605-d64ce87f3ad6.png">

Perhaps we could update everything to v8.0 on https://www.pivotaltracker.com/story/show/179040976